packages/server/customer-os-api/graph/resolver/meeting.resolvers_it_test.go

```diff
@@ -119,6 +120,80 @@ func TestMutationResolver_Meeting(t *testing.T) {
 	require.Equal(t, "test-recording-id", meeting.Meeting_Update.Recording)
 }
 
+func TestMutationResolver_MergeContactsWithMeetings(t *testing.T) {
+	ctx := context.TODO()
+	defer tearDownTestCase(ctx)(t)
+	neo4jt.CreateTenant(ctx, driver, tenantName)
+	neo4jt.CreateDefaultUserWithId(ctx, driver, tenantName, "test_user_id")
+	neo4jt.CreateContactWithId(ctx, driver, tenantName, "test_contact_id_1", entity.ContactEntity{
```

Comment:
please extract all the ids as a vars. it's easy to track in code than the actual string
contactId1 := "test_contact_id_1"